src/main/resources/application.yaml

```diff
@@ -33,6 +37,10 @@ spring:
       password: rootpass
       username: rootuser
       authentication-database: admin
+  jpa:
+    hibernate:
+      ddl-auto: update
```

Comment:
should be `ddl-auto: validate` when the migrations will be applied.

Reply:
Fixed